File: sample/python/math_utils.py
# math_utils.py - 기본 수학 유틸리티 함수들

def add(a, b):
    """두 숫자를 더합니다"""
    return a + b


def multiply(a, b):
    """두 숫자를 곱합니다"""
    return a * b


def sum_list(numbers):
    """배열의 합계를 계산합니다"""
    total = 0
    for n in numbers:
        total = add(total, n)  # add 함수 사용
    return total

File: sample/csharp/MathUtils.cs
using System;

namespace ListCodeUsagesSample
{
    /// <summary>
    /// 기본 수학 유틸리티 클래스
    /// </summary>
    public static class MathUtils
    {
        /// <summary>
        /// 두 숫자를 더합니다
        /// </summary>
        public static int Add(int a, int b)
        {
            return a + b;
        }

        /// <summary>
        /// 두 숫자를 곱합니다
        /// </summary>
        public static int Multiply(int a, int b)
        {
            return a * b;
        }

        /// <summary>
        /// 배열의 합계를 계산합니다
        /// </summary>
        public static int Sum(int[] numbers)
        {
            int total = 0;
            foreach (var n in numbers)
            {
                total = Add(total, n); // Add 메서드 사용
            }
            return total;
        }
    }
}
